### ISSUE TITLE
v9 compatibility

### ISSUE THREAD

**Issue by mcglincy** [mcglincy/dungeondraw-foundry-vtt#47]
Tracking bug for issues related to v9.

First issue: black screen for foundry and a console error:

```
foundry.js:326 TypeError: Failed to render WebGL canvas: Cannot read properties of undefined (reading 'dungeon')
    at Canvas._createGroups (foundry.js:19340)
    at Canvas.initialize (foundry.js:19266)
    at Game.initializeCanvas (foundry.js:5479)
    at async Game.setupGame (foundry.js:5353)
    at async Game._initializeGameView (foundry.js:6291)
    at async Game.initialize (foundry.js:5249)
```

**Comment by mcglincy** [comment]
Fixed in dungeondraw.js by 

```
    CONFIG.Canvas.layers.dungeon = {
      layerClass: DungeonLayer,
      group: "primary"
    };
```

Next error:
```
renderer.js:339 Uncaught (in promise) TypeError: Cannot read properties of null (reading 'getNumGeometries')
    at Dr (renderer.js:339)
    at Cr (renderer.js:129)
    at renderer.js:10
```

**Comment by mcglincy** [comment]
Fixed in 0.13.0.
